spec/configuration/prison_data_spec.rb

```diff
@@ -20,18 +20,48 @@
   context "enabled prisons" do
     let(:enabled) { Prison.enabled }
 
+    it "currently number between 95 and 105" do
+      expect(enabled.count).to be_within(5).of(100)
+    end
+
     it "each has a nomis_id" do
       expect(enabled.any?{ |p| p.nomis_id.blank? }).to be_falsey
     end
 
-    it "each unbookable date for each prison should be a valid date" do
-      unbookable = enabled.map(&:unbookable).flatten
-      expect(unbookable.map(&:class).all?{ |p| p == Date }).to be_truthy
-    end
+    context 'unbookable dates' do
+      it "are all valid dates" do
+        unbookable = enabled.flat_map(&:unbookable)
+        expect(unbookable.map(&:class).all?{ |p| p == Date }).to be_truthy
+      end
 
-    it "each unbookable date for each prison should be a unique" do
-      enabled.each do |prison|
-        expect(prison.unbookable.size).to eq(prison.unbookable.uniq.size)
+      it "are all unique" do
+        enabled.each do |prison|
+          expect(prison.unbookable.size).to eq(prison.unbookable.uniq.size)
+        end
+      end
+
+      context 'do not conflict with anomalous dates' do
+        describe 'and breaks the spec run if there are conflicts' do
+          subject { Prison.find('Rochester') }
+          before do
+            allow(subject).to receive(:slot_anomalies).
+              and_return(Time.zone.today => [])
+            allow(subject).to receive(:unbookable).
+              and_return([Time.zone.today, Time.zone.tomorrow])
+          end
+
+          specify do
+            expect(subject.anomalous_dates - subject.unbookable_dates).
+              not_to eq(subject.anomalous_dates)
```

Comment:
I think logic operators might make more sense here:

```ruby
expect(subject.anomalous_dates & subject.unbookable_dates).to be_empty
```

(I assume there's no ready-made `expect(a).not_to intersect(b)` matcher, though that would be nice.)